src/SprykerEco/Zed/Payone/Business/Api/Response/Container/SessionResponseContainer.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Api\Response\Container;
+
+class SessionResponseContainer extends AbstractResponseContainer
```

Comment:
* Maybe better name will be `KlarnaStartSessionResponseContainer` ?

Reply:
You are right